test/SparseGenMatProd.cpp

```diff
@@ -53,3 +77,26 @@ TEMPLATE_TEST_CASE("matrix operations", "[SparseGenMatProd]", float, double)
     INFO("The accesor operator must produce the same element as in eigen")
     REQUIRE(mat1.coeff(45, 22) == sparse1(45, 22));
 }
+
+TEMPLATE_TEST_CASE("matrix operations [100000000x100000000]", "[SparseGenMatProd]", float, double)
```

Comment:
@JensWehner I cannot go any higher before I get alloc errors, OOM action etc. Is this good enough?

Reply:
this is way too high :) 1000x1000 is more than enough